src/biunilm/decode_seq2seq.py

```diff
@@ -91,6 +91,12 @@ def main():
                         help="Batch size for decoding.")
     parser.add_argument('--beam_size', type=int, default=1,
                         help="Beam size for searching")
+    parser.add_argument('--temperature', type=float, default=1,
```

Comment:
remembered last night this needs to default to 0 to maintain the same behavior as the old code when disabled, I will update the PR today when I get the chance